client-runtime/src/main/java/com/microsoft/rest/v2/http/HttpHeaders.java

```diff
@@ -33,6 +49,18 @@ public HttpHeaders add(String headerName, String headerValue) {
         return this;
     }
 
+    /**
+     * Get the header value for the provided header name. If the header name isn't found, then null
+     * will be returned.
+     * @param headerName The name of the header to look for.
+     * @return The String value of the header, or null if the header isn't found.
+     */
+    public String get(String headerName) {
+        final String headerKey = headerName.toLowerCase();
+        final HttpHeader header = headers.get(headerKey);
+        return header == null ? null : header.getValue();
```

Comment:
I find it really frustrating when I know that a collection has a single value in it, but there's no easy way to get the single value without iterating through the collection. For this reason, I've left the original get() method (renamed to getValue()) and I've added getValues() that returns a String[].